docs/docs/ingest/optimize.md

```diff
@@ -15,7 +15,7 @@ into your RAG application. This is advantageous for the following reasons:
 
 ### Text
 
-Using [Markdown](https://www.markdownguide.org/) as the standard format for content
+Using [Markdown](https://www.markdownguide.org/) as the standard format
 is a good choice for most text-based content. LLMs can reason with Markdown,
```

Comment:
```suggestion
[Markdown](https://www.markdownguide.org/) works well as the standard format for most text-based content. LLMs can reason with Markdown,
```

docsy nit to avoid the "using" gerund phrase